andela/wg-bulls
# #157589611 Reset Muscle Cache on Delete

#### What does this PR do?
Exercise cache when deleting muscles
#### Description of Task completed
The cache should be reset after a muscle has been deleted such that the deleted muscle in not visible from the exercises page.
Files changed include:
* `wger/exercises/signals.py`
#### How should this be manually tested?
Log in to the application as an admin, visit the muscle menu under the exercise tab https://wg-bulls.herokuapp.com/en/exercise/muscle/admin-overview/
Click on the add muscle button and add a muscle name.
Visit the add exercise tab and add a new exercise focusing on the new muscle added.
Go back to the muscles menu and delete the recently added muscle and verify that the muscle no-longer included under the exercise menu.
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/157589611
